diff --git a/drivers/gpu/drm/i915/i915_debugfs.c b/drivers/gpu/drm/i915/i915_debugfs.c
index 5c45e9e..1e57bce 100644
--- a/drivers/gpu/drm/i915/i915_debugfs.c
+++ b/drivers/gpu/drm/i915/i915_debugfs.c
@@ -1714,6 +1714,7 @@ static int i915_edp_psr_status(struct seq_file *m, void *data)
 	return 0;
 }
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,8,0)
 static int i915_energy_uJ(struct seq_file *m, void *data)
 {
 	struct drm_info_node *node = m->private;
@@ -2235,6 +2236,7 @@ static int display_crc_ctl_parse_pipe(const char *buf, enum pipe *pipe)
 
 	return 0;
 }
+#endif
 
 static int
 display_crc_ctl_parse_source(const char *buf, enum intel_pipe_crc_source *s)
@@ -2818,7 +2820,9 @@ static struct drm_info_list i915_debugfs_list[] = {
 	{"i915_dpio", i915_dpio_info, 0},
 	{"i915_llc", i915_llc, 0},
 	{"i915_edp_psr_status", i915_edp_psr_status, 0},
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,8,0)
 	{"i915_energy_uJ", i915_energy_uJ, 0},
+#endif
 	{"i915_pc8_status", i915_pc8_status, 0},
 };
 #define I915_DEBUGFS_ENTRIES ARRAY_SIZE(i915_debugfs_list)
